bgpd/bgp_route.c

```diff
@@ -3157,17 +3158,11 @@ int bgp_update(struct peer *peer, struct prefix *p, uint32_t addpath_id,
 
```

Comment:
below code is dead code.

	memset(&new_attr, 0, sizeof(struct attr));
	new_attr.label_index = BGP_INVALID_LABEL_INDEX;
	new_attr.label = MPLS_INVALID_LABEL;

Is this init required as new_attr is not being used before shallow copy and shallow copy should have the values set to right values?

Reply:
No probably not. Didn't write it so I'm not changing it. Will come back to it later.

Reply:
Sure. It is just a dead code does not harm for now.